sorts/bubble_sort.py

```diff
@@ -28,9 +31,9 @@ def bubble_sort(collection):
     True
     """
     length = len(collection)
-    for i in range(length - 1):
+    for i in reversed(range(length)):
```

Comment:
Is calling `reversed` is more efficient? I don't think so. Did you do some benchmarks?

Reply:
`reversed` has to be a bit better since the Python `heapq.py` module uses it.
It also improves code readability. @L3str4nge 

Source: https://www.python.org/dev/peps/pep-0322/

Reply:
I know but it would be good to know if algorithms performance is better too :) Could you perform any benchmarks before and after refactor?

Reply:
Benchmarked with the `timeit` module:

1000 items:

```
Before: 
0.10902250000071945

After: 
0.11019620000297436
```
5000 items:

```
Before: 
2.8898907999973744

After: 
2.844012000001385
```
10000 items:

```
Before: 
12.34897049999563

After: 
11.655540399995516
```
I tested a bit more locally and sometimes refactored one is better and sometimes it's the original.